api/routes/stacks.py

```diff
@@ -0,0 +1,77 @@
+from fastapi import APIRouter, Depends, status, Response
+from api.api_models import stacks as stack_schemas
+from utils.permissions import is_admin
+
+from db.database import get_db
+from sqlalchemy.orm import Session
+from db.models.users import User, Stack
+
+
+
+
+stack_router = APIRouter(tags=["Stacks"], prefix="/stacks")
+
+# Return all stacks
+@stack_router.get("/", response_model=list[stack_schemas.Stacks])
+async def list_stacks(db:Session = Depends(get_db), page:int = 1, limit: int = 100):
+	# calculate skip value
+	page = page if page >= 1 else 1
+	skip = (page - 1) * limit
+
+	stacks = db.query(Stack).offset(skip).limit(limit).all()
+	return stacks
+
+
+# create a new Stack
+@stack_router.post("/", response_model=stack_schemas.Stacks, status_code=status.HTTP_201_CREATED)
+async def create_stack(stack:stack_schemas.StackCreate, user:User = Depends(is_admin), db: Session = Depends(get_db)):
+	new_stack = Stack(**dict(stack))
+
+	db.add(new_stack)
+	db.commit()
+	db.refresh(new_stack)
+
+	return new_stack
+
+# read stack with id
+@stack_router.get("/{stack_id}", response_model=stack_schemas.Stacks)
+async def read_stack(stack_id: int, db: Session = Depends(get_db)):
+	stack = db.query(Stack).filter(Stack.id == stack_id).first()
```

Comment:
If we have a primary key on the table, we can use the `session.get(Stack, {stack_id})`

Reply:
this is what is being used in the docs, havent found any use of a .get so not sure if that is possible 
https://fastapi.tiangolo.com/tutorial/sql-databases/#read-data

Reply:
this is what is being used in the docs, haven't found any use of a .get so not sure if that is possible 
https://fastapi.tiangolo.com/tutorial/sql-databases/#read-data

Reply:
Alright thanks.